README.md

```diff
@@ -1,2 +1,221 @@
-# utilihive-backend-stub-sftp-server
-SFTP backend stub
+# Utilihive SFTP Server Stub
+
+SFTP Server Stub runs an in-memory SFTP server while your tests are running.
+It uses the SFTP server of the
+[Apache SSHD](http://mina.apache.org/sshd-project/index.html) project.
+It is made to simplify functional testing of [flows](https://docs.utilihive.io/utilihive-integration/core-concepts/flows)
+written for [Utilihive flow-server](https://docs.utilihive.io/utilihive-integration/core-concepts/flow-server)
+by providing a mock/stub of a SFTP server and basic functions for manipulating files on it.
+However, it is not dependent on Utilihive platform and can be used independently.
+
+SFTP Server Stub is published under the
+[MIT license](http://opensource.org/licenses/MIT) and is originally based on Stefan Birkner's
+[Fake SFTP Server Lambda](https://github.com/stefanbirkner/fake-sftp-server-lambda),
+although it was rewritten to Kotlin. It requires at least JDK 11.
+
+
+## Installation
+
+SFTP Server Stub is available from
+[Maven Central](https://search.maven.org/#search|ga|1|utilihive-backend-stub-sftp).
+
+    <dependency>
+      <groupId>com.greenbird.utilihive.stubs</groupId>
+      <artifactId>utilihive-backend-stub-sftp-server</artifactId>
+      <version>1.0.0</version>
```

Comment:
Can we have the current release version auto-generated here? Suspecting github supports this. Maybe we need a 1.0.0 release first?

Reply:
I have added and tested the generation using maven-resources-plugin - it copies the readme file from doc/ directory and maven-release-plugin makes sure that during release:prepare phase, the copied file with the release version is added to Git (if we don't add it to Git, it won't be displayed on the front page of Github). I am only committing the file in <preparationGoals> as we don't won't to commit the next snapshot version of the readme, only the previous release.